test/testinput/buoy_wmoBUFR2ioda.yaml

```diff
@@ -37,117 +37,117 @@ observations:
             mnemonic: CLONH
           stationElevation:
             mnemonic: HSAWS
-          temperatureAir:
+          airTemperature:
             mnemonic: TMDB
-          temperatureDewpoint:
+          dewpointTemperature:
             mnemonic: TMDP
           windDirection:
             mnemonic: WDIR
           windSpeed:
             mnemonic: WSPD
-          pressureStation:
+          stationPressure:
             mnemonic: PRES
-          meanSeaLevelPressure:
+          pressureReducedToMeanSeaLevel:
             mnemonic: PMSL
-          temperatureSeaSurface:
+          seaSurfaceTemperature:
             mnemonic: SST0
 
     ioda:
       backend: netcdf
       obsdataout: "./testrun/buoy_wmo_multi.nc"
 
       dimensions:
-        - name: "nlocs"
+        - name: "Location"
           size: variables/latitude.nrows
 
       variables:
         - name: "MetaData/dateTime"
           source: variables/timestamp
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "dateTime"
           units: "seconds since 1970-01-01T00:00:00Z"
 
         - name: "MetaData/latitude"
           source: variables/latitude
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Latitude"
           units: "degrees_north"
 
         - name: "MetaData/longitude"
           source: variables/longitude
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Longitude"
           units: "degrees_east"
 
-        - name: "MetaData/station_id"
+        - name: "MetaData/stationIdentification"
           source: variables/stationIdentification
-          dimensions: [ "nlocs" ]
-          longName: "Station Identification"
+          dimensions: [ "Location" ]
+          longName: "Station Identifier"
           units: "none"
 
         - name: "MetaData/stationLongName"
           source: variables/stationLongName
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Station long name"
           units: "none"
 
-        - name: "MetaData/station_elevation"
+        - name: "MetaData/stationElevation"
           source: variables/stationElevation
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
           longName: "Station Elevation"
           units: "m"
 
         - name: "MetaData/height"
           source: variables/stationElevation
-          dimensions: [ "nlocs" ]
+          dimensions: [ "Location" ]
```

Comment:
This and the previous variable are identical? Is that correct? I guess so since a buoy should be `0`?

Reply:
@CoryMartin-NOAA   Agree this is a total HACK, but there is no choice really.  All of the treatment of conventional data for pressure (and probably temp/wind/moisture) require a station elevation and height of ob separately.  Buoy data rarely reports both.  This hack provides a way to fill the height variable which should generally be a meter or so higher than so-called station elevation.  This is a kinda problem with Ron's converter in general.  If the data are lacking in the first place, then a straight bufr-to-ioda is often useless.  Very frustrating that BUFR allows for rich content, but countries making those data have to fill things in (and they don't do enough of it).